Crash when starting app [1h-2h]

ClassCastException

Caused by: java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.Long
                                                                             at android.app.SharedPreferencesImpl.getLong(SharedPreferencesImpl.java:249)
                                                                             at ai.elimu.appstore.MainActivity.onStart(MainActivity.java:94)

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,5 +1,5 @@
 [versions]
-elimuModel = "model-2.0.83"
+elimuModel = "model-2.0.84"
 material = "1.5.0"
 retrofit = "2.11.0"
 junit = "4.13.2"

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,5 +1,5 @@
 [versions]
-elimuModel = "model-2.0.86"
+elimuModel = "model-2.0.87"
 material = "1.5.0"
 retrofit = "2.11.0"
 junit = "4.13.2"

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,5 +1,5 @@
 [versions]
-elimuModel = "model-2.0.87"
+elimuModel = "model-2.0.88"
 material = "1.5.0"
 retrofit = "2.11.0"
 junit = "4.13.2"

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,5 +1,5 @@
 [versions]
-elimuModel = "model-2.0.84"
+elimuModel = "model-2.0.85"
 material = "1.5.0"
 retrofit = "2.11.0"
 junit = "4.13.2"

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,5 +1,5 @@
 [versions]
-elimuModel = "model-2.0.85"
+elimuModel = "model-2.0.86"
 material = "1.5.0"
 retrofit = "2.11.0"
 junit = "4.13.2"

